packages/integration-tests/src/views/storagePage.ts

```diff
@@ -24,4 +24,18 @@ export const storagePage = {
         expect(hostDisk).to.contain('2');
       });
   },
+   getSkipFormattingCheckbox: () => {
+    return cy.get('input[id="select-formatted-0"]');
+  },
+  validateSkipFormattingDisks: (
+    numWorkers: number = Cypress.env('NUM_WORKERS'),
+  ) => {
+    cy.get(Cypress.env('skipFormattingDataLabel'))
+      .should('have.length', numWorkers)
+    cy.get('input[type="checkbox"]').should('not.be.checked')
```

Comment:
Please use `getSkipFormattingCheckbox` instead, passing the index of the disk.
Otherwise, the code only works for cases with 1 disk.